module-2/backend/week_one.md

```diff
@@ -4,29 +4,73 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
+1. List the five common HTTP verbs and what the purpose is of each verb.
+  * GET: 'get' the stuff on the page
+  * PUT: update data
+  * POST: add new data
+  * DELETE: remove data
+  * HEAD: just get head information from the response
+2. What is Sinatra?
+  * Sinatra is a Ruby framework
+4. What is MVC?
+  * MVC stands for Model View Controller which is a way of seperating responsibilities
+5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  * To make things easier to understand/remember and to reduce error
+6. What types of variables are accessible in our view templates without explicitly passing them?
+  * COME BACK
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
     erb :index
   end
+  ```
+  
+  ```ruby
+  get '/horses' do
+    @count = 1
+    erb :index
+  end
   ```
 
-8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
-9. What's the purpose of ERB?
-10. Why do I need a development AND test database?
-11. What is CRUD and why is it important?
-12. What does HTTP stand for? 
-13. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
-14. What's an ORM?
-15. What's the most commonly used ORM in ruby (Sinatra & Rails)?
-16. Let's say we have an application with restaurants. There are seven verb + path combinations necessary to provide full CRUD functionality for our restaurant application. List each of the seven combinations, and explain what each is for.
-17. What's a migration? 
-18. When you create a migration, does it automatically modify your database?
-19. How does a model relate to a database?
-20. What is the difference between `#new` and `#create`?
+8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+  * After internet research...
+  
+  ```ruby
+  get '/horses' do
+    @count = 1
+    render locals: { name: "Mr. Ed" }
+    erb :index

```

Comment:
would just add onto the `erb: index` line `erb: index, locals: {name: "Mr. Ed"}`